docs/source/data-model.rst

```diff
@@ -322,6 +350,18 @@ Minimal nontrivial valid example:
     'spec': 'SOME_SPEC',
     'uid': '272132cf-564f-428f-bf6b-149ee4287024'}
 
+Typical example:
```

Comment:
Do we want to delete the version just above?

Reply:
I see your point, but we have "Minimal" and "Typical" examples in every other case, and it might help to illustrate that `resource_kwargs` has not requires keys.